### PR TITLE
MSZ strong motion replacement

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Closed location 20 and opened location 21
- Removed Kinemetrics Obsidian 4X (695) and Kinemetrics FBA-ES-T episensor (2266) at location 20(HN-ENZ)
- Replaced with Nanometrics TitanSMA (002025) at location 21(HN-12Z) with azimuth 240

Let me know if there's anything I missed in all this.

### CODE REVIEW

**Review by CBurton90 — DISMISSED** [review]

**Review by mnaguit — CHANGES_REQUESTED** [review]
- do we need to update the connections? is this a sensor/datalogger to recorder swap? 
- shall we add a firmware detail for EQR120? `v4.5.22`?

File: install/connections.csv [hunk 1]
```diff
@@ -543,7 +543,8 @@ MSWZ,10,Moikau Station,,,2003-08-28T20:55:00Z,9999-01-01T00:00:00Z
 MSZ,10,Milford Sound,,0,2005-07-19T20:00:00Z,9999-01-01T00:00:00Z
 MSZ,10,Milford Sound,Short Period,,1990-05-28T11:33:00Z,2005-06-15T04:06:00Z
 MSZ,20,Milford Sound,,3,2005-07-19T20:00:00Z,2017-05-23T04:15:00Z
-MSZ,20,Milford Sound,Strong Motion,,2017-05-23T04:15:01Z,9999-01-01T00:00:00Z
+MSZ,20,Milford Sound,Strong Motion,,2017-05-23T04:15:01Z,2023-06-13T23:17:00Z
+MSZ,21,Milford Sound,Strong Motion,,2023-06-14T03:35:00Z,9999-01-01T00:00:00Z
```
Comment (CBurton90) [inline install/connections.csv:547]: Ah yes, as a recorder has gone in I think line 547 isn't needed.

Comment (Thomas-Benson) [reply install/connections.csv]: Line eliminated. anything else I missed here or is this good?

File: install/firmware.csv [hunk 2]
```diff
@@ -543,6 +543,7 @@ Nanometrics,TitanSMA,001898,4.5.22,2022-07-13T00:30:00Z,9999-01-01T00:00:00Z,
 Nanometrics,TitanSMA,001912,4.3.28,2022-10-20T02:28:00Z,9999-01-01T00:00:00Z,
 Nanometrics,TitanSMA,001913,4.5.22,2023-01-29T01:27:00Z,9999-01-01T00:00:00Z,
 Nanometrics,TitanSMA,001915,4.5.22,2022-09-06T21:37:00Z,9999-01-01T00:00:00Z,
+Nanometrics,TitanSMA,002026,4.5.22,2022-06-14T03:35:00Z,9999-01-01T00:00:00Z,
```
Comment (mnaguit) [inline install/firmware.csv:546]: opening year is `2023`

**Review by mnaguit — CHANGES_REQUESTED** [review]

**Comment by Thomas-Benson** [comment]
- Added TitanSMA to install/firmware.csv
- Removed unneeded connection in install/connections.csv

**Comment by mnaguit** [comment]
Thanks @Thomas-Benson
Noting that  
- The datalogger that is still open (`Q330S/6`) is connected to the sensor at LC 10
- Sensor/datalogger(`FBA/Obsidian`) at LC 20 now closed.
- Standalone recorder (`TitanSMA`) opens at LC 21

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**